assets/js/util.js

```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
+        if($('#originalText').height() < originalTextScrollHeight) {
+            $('#originalText').css('overflow-y', 'scroll');
```

Comment:
> Why can't this situation be solved with simply a `overflow: y-scroll`?

Um, I had tried that! As soon as `synchronizeTextareaHeights` method is called, the `CSS` for this div changes. And thus, I started with this approach of checking if the `textarea` height falls in the range for scrolling!